src/routes.json

```diff
@@ -218,5 +218,33 @@
         "pattern": "^/explore/studios/?$",
         "routeAlias": "/explore(?!/ajax)",
         "redirect": "/explore/studios/all"
+    },
+    {
+        "name": "microworld-art",
+        "pattern": "^/microworlds/art",
```

Comment:
Whoops I meant in the `routeAlias` we don't need that, we still need it in the `route`. So, it's not needed in each `routeAlias`, like below.